agent/src/podman/podman_runtime_config.rs

```diff
@@ -53,3 +55,56 @@ impl From<TryFromWorkloadSpecError> for WorkloadError {
         WorkloadError::StartError(value.0)
     }
 }
+
+#[derive(Debug, serde::Deserialize)]
+pub struct Mount {
+    #[serde(default)]
+    pub destination: Option<String>,
+    #[serde(default)]
+    pub options: Option<Vec<String>>,
+    #[serde(default)]
+    pub source: Option<String>,
```

Comment:
We could add some aliases:
```suggestion
    #[serde(default)]
    #[serde(alias = "dst")]
    pub destination: Option<String>,
    #[serde(default)]
    pub options: Option<Vec<String>>,
    #[serde(default)]
    #[serde(alias = "src")]
    pub source: Option<String>,
```

Reply:
I have added alias for `source` and `destination`. I also have added a rename for `_type`.